### ISSUE TITLE
Static vs dynamic URI references

### ISSUE THREAD

**Issue by ndw** [xproc/3.0-specification#686]
I propose to amend section 7.1 to make it clear that the `doc.xml` read statically to compute `$B` is not guaranteed to be the same `doc.xml` that's ready dynamically!

**Comment by xml-project** [comment]
Yes, we should say this somewhere because this is a point where we differ from XSLT (and XPath?).

**Comment by ndw** [comment]
In Section 4.1, we say:

> Version 3.0 of XProc does not require implementations to guarantee that multiple attempts to dereference the same URI always produce consistent results.

I will propose that we change "consistent" to "the same" in a subsequent editorial PR.

**Comment by xml-project** [comment]
+1 "consistent" was also on my list.
